pkgs/data/fonts/tamsyn/default.nix

```
@@ -0,0 +1,49 @@
+{ stdenv, fetchurl, mkfontdir, mkfontscale }:
+
+let
+  version = "1.11"; 
+in stdenv.mkDerivation rec {
```

```suggestion
in stdenv.mkDerivation {
```
If you are going to have version in the let block, you probably don't need a recursive attrset.